tests/page-templates/19-navtiming/04-no-resend-after-complete.js

```diff
@@ -0,0 +1,18 @@
+/*eslint-env mocha*/
+/*global BOOMR_test,assert*/
+
+describe("e2e/19-navtiming/04-no-resend-after-complete", function() {
+	var tf = BOOMR.plugins.TestFramework;
+	var t = BOOMR_test;
+
+	it("Should only send navigation timing information on second beacon (if NavigationTiming is supported)", function() {
+//		if (t.isNavigationTimingSupported()) {
```

Comment:
Was this commented out for debugging?

In the else case, you can do `this.skip()`